tutorials/using-cloud-vpn-with-checkpoint/index.md

```diff
@@ -0,0 +1,383 @@
+4. The VTIs show in the topology.
+    **Note**: The **Edit Topology** window lists the members of a VTI on the same line if these criteria match:
+
+  * Remote peer name
+  * Remote IP address
+  * Interface name
+
+5. Configure the VTI VIP in the Topology tab. Click **OK**.
+6. From **VPN Domain**, select **Manually Defined** > **Empty_Group**.
+
+
+![alt_text](Image_15.PNG)
+
+![alt_text](Image_16.PNG)
+
+**Step 6**. Create a star community.
+
+1. Open **SmartConsole** > **Security Policies** > **Access Tools** > **VPN Communities**.
+2. Click **Star Community**.
+3. Enter an **Object Name** for the VPN Community.
+4. In the **Center Gateways** area, click the plus sign to add a Check Point Security Gateway object for the center of the community.
+5. In the **Satellite Gateways** area, click the plus sign to add the GCP gateway object.
+
+![alt_text](Image_17.PNG)
+
+**Step 7**. Configure these ciphers for IKEv2.
+
+Go to **Encryption** and change the Phase 1 and Phase 2 properties according what is specified within the Cipher configuration settings on page 3)
+**Note:** You must select Perfect Forward Secrecy (Phase 2).
+
+This example refers to IKEv2 specifically. You can also use IKEv1 in this scenario.
+
+![alt_text](Image_18.PNG)
+
+**Step 8**. Go to the **Advanced tab**. You can modify the more advanced settings for Phase 1 Phase 2 there.
+
+![alt_text](Image_19.PNG)
+
+**Step 9**. Setup for BGP Deployment
+
+**Virtual Tunnel Interface and Initial BGP Setup**
+
+Connect with SSH to your Security Gateway. If you are using the none default shell, change to clish. Run: `clish`
+Run the commands below replacing variables surrounded by { } with your values:
+
+    set AS {CP Security Gateway - ASN}
+    set bgp external remote-as { GCP-ASN } on
+    set bgp external remote-as { GCP-ASN } peer { TUN-INSIDE- GCP } on
+    set bgp external remote-as { GCP-ASN } peer { TUN-INSIDE- GCP } as-override on
+    set bgp external remote-as { GCP-ASN } peer { TUN-INSIDE- GCP } holdtime 30
+    set bgp external remote-as { GCP-ASN } peer { TUN-INSIDE- GCP } keepalive 10
+    set inbound-route- filter bgp-policy 512 based-on- as as { GCP-ASN }  on
+    set inbound-route- filter bgp-policy 512 accept-all- ipv4
+    set route-redistribution to bgp-as { GCP-ASN }  from interface {Redistributed from specific interface } on
+
+Example:
+
+    set as 65002
+    set bgp external remote-as 65000 on
+    set bgp external remote-as 65000 peer 169.254.0.1 on
+    set bgp external remote-as 65000 peer 169.254.0.1 as-override on
+    set bgp external remote-as 65000 peer 169.254.0.1 holdtime 30
+    set bgp external remote-as 65000 peer 169.254.0.1 keepalive 10
+    set inbound-route- filter bgp-policy 512 based-on- as as 65000 on
+    set inbound-route- filter bgp-policy 512 accept-all- ipv4
+    set route-redistribution to bgp-as 65000 from interface eth1 on
+
+**Step 10**. Configure Directional Rules.
+
+Adding Directional Rules for a Route-Based Scenario
```

Comment:
Either omit "Add Directional ..." sentence and add to Step 10 (Configure Directional Rules for a Route Based Scenario" 
or
Add to note below.